integration-tests/Dockerfile_arch

```diff
@@ -1,7 +1,8 @@
-FROM base/archlinux
+FROM archlinux:base
 MAINTAINER @siddharthist
 
-RUN pacman -Sy --noconfirm systemd-sysvcompat
+# systemd-sysvcompat is already installed
+# RUN pacman -Sy --noconfirm systemd-sysvcompat
```

Comment:
Please either remove these, or leave a comment elaborating more about the context of how come they're present but commented out?

Reply:
I have removed this two lines.